x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -260,56 +315,16 @@ describe('get mapbox color expression', () => {
       test('should return null when color ramp is not provided', async () => {
         const dynamicStyleOptions = {
           type: COLOR_MAP_TYPE.ORDINAL,
-          field: {
-            name: 'myField',
-          },
         };
         const colorProperty = makeProperty(dynamicStyleOptions);
         expect(colorProperty._getMbColor()).toBeNull();
       });
-
-      test('should return mapbox expression for color ramp', async () => {
```

Comment:
This test case should not be removed. The `get mapbox color expression` describe block carefully tests each case and branch. Its great to add a new test to test some other path, but this test is an important part of `get mapbox color expression`

Reply:
discussed in person. removing. can follow up with new test suite against public API in separate PR